packages/contracts/package.json

```diff
@@ -37,18 +37,19 @@
     "url": "https://github.com/aragon/osx/issues"
   },
   "homepage": "https://github.com/aragon/osx#readme",
+  "dependencies": {
+    "@ensdomains/ens-contracts": "0.0.11",
```

Comment:
Since `ENSSubdomainRegistrar` has an `ENS` member that we import from `@ensdomains/ens-contracts`, it is a real `dependencies` and not only a `devDependencies`.

Reply:
Nevermind, we don't have these integration tests on the `main` branch yet. They will end up there once we merge `develop` into `main` (probably when we publish `1.4.0`).